### PR TITLE
Fixing CLI references

### PR DESCRIPTION
**Pull request by agraebe**
Partially closing #952

### CODE REVIEW

File: src/pages/references/stacks-cli.md [hunk 1]
```diff
@@ -27,6 +28,30 @@ You must have [Node.js](https://nodejs.org/en/download/) v12 or higher (v14 reco
 
 To install the command line, do the following:
 
+### Step 1: [Download or `git clone` the command line repository code](https://github.com/blockstack/stacks.js/tree/master/packages/cli)
```
Comment (yknl) [inline src/pages/references/stacks-cli.md:31]: I think we should just delete the clone repo instructions and use `npm install -g @stacks/cli` here.

File: src/pages/understand-stacks/install-api.md [hunk 2]
```diff
@@ -24,8 +24,8 @@ $ cd blockstack-core/
 $ git checkout stacks-1.0
 $ pip install .
 $ pip install -r api/requirements.txt
-$ blockstack setup_wallet
-$ blockstack api start
+$ stx setup_wallet
```
Comment (yknl) [inline src/pages/understand-stacks/install-api.md:27]: This is very outdated documentation and no longer relevant. We should just delete the whole thing.

**Review by yknl — APPROVED** [review]

File: src/pages/understand-stacks/install-api.md (hunk 2 above)
Comment (agraebe) [reply src/pages/understand-stacks/install-api.md:27]: can we delete the whole file?

File: src/pages/references/stacks-cli.md (hunk 1 above)
Comment (agraebe) [reply src/pages/references/stacks-cli.md:31]: good point! I didn't look closely enough. will us npm instead

File: src/pages/understand-stacks/install-api.md (hunk 2 above)
Comment (yknl) [reply src/pages/understand-stacks/install-api.md:27]: These instructions are for the 1.0 blockstack-core. Either we remove the whole page, or add a notice at the top that this is out of date and remove it after 2.0 mainnet.

Comment (agraebe) [reply src/pages/understand-stacks/install-api.md:29]: added a callout

**Comment by aulneau** [comment]
Please rebase and push to fix the build issue.

**Pull request merged**